style.css

```diff
@@ -312,6 +313,83 @@ i:hover {
     color: var(--text-col);
 }
 
+/* --cart-page--- */
+.cart-container {
+    padding: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.prod-in-cart{
+    display:flex;
+    margin:1rem;
+    background-color: var(--white-color);
+    box-shadow: 2px 2px 5px 5px var(--light-grey);
+    height: max-content;
+}
+.prod-detail{
+    margin:0 1rem 0.2rem 1rem;
+}
+.prod-in-cart .text-lin-through{
+    text-decoration: line-through;
+}
+.prod-in-cart .grey-text{
+    color:var(--little-dark-grey);
+}
+.prod-in-cart img {
+    width: 180px;
+    height:max-content;
+}
+.quant{
+    display:flex;
+    align-items: center;
+}
+.quant i{
+    border:1px solid var(--little-light-grey);
+    border-radius: 50%;
+    padding:0.5rem;
+}
+.quant input{
+    width:10%;
+    margin:0.5rem;
```

Comment:
If the value is less than 10px, use can use px directly instead of rem

Reply:
Yes, but here it is 50px just because of it I used rem